crates/editor_tabs/src/colors.rs

```diff
@@ -0,0 +1,15 @@
+/// Colors used in editor
+use bevy_egui::egui::{Color32, Stroke};
```

Comment:
I think color, sizing and typography would live better in the UI crate, as they are ui elements. If this crate is the one responsible for the application ui, maybe call this editor_ui and the other editor_ui_extensions/editor_ui_components?

Reply:
Yes, this is undoubtedly better located in editor_ui crate. However, these colours are referenced by editor_tabs and you need to get rid of that. However, the PR is already quite large, so I decided to stop with just shuffling the code for now. And the improvement of the separations will be in multiple PRs